pylint/testutils/lint_module_test.py

```diff
@@ -138,16 +138,14 @@ def _open_source_file(self):
         return open(self._test_file.source, encoding="utf8")
 
     def _get_expected(self):
-        with self._open_source_file() as fobj:
-            expected_msgs = self.get_expected_messages(fobj)
-
-        if expected_msgs:
-            with self._open_expected_file() as fobj:
-                expected_output_lines = [
-                    OutputLine.from_csv(row) for row in csv.reader(fobj, "test")
-                ]
-        else:
-            expected_output_lines = []
+        with self._open_source_file() as f:
+            expected_msgs = self.get_expected_messages(f)
+        if not expected_msgs:
+            return expected_msgs, []
```

Comment:
Is it equivalent to `return [], []`?

Reply:
Yeah I thought so too but this is a `collections.Counter()` not a list. I broke the first test on the `.py` file trying to refactor, so I reverted. I think `return collections.Counter(), []` would work though.

Reply:
Maybe returning `Counter()` is clearer in the sense that it clearly indicates that it is empty. What do you think about it?

Reply:
I agree, let's do this :)